org.aliflang.lang.yml

```diff
@@ -0,0 +1,136 @@
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: alifstudio
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --share=network
+  - --env=PATH=$PATH:/usr/bin:/bin:/app/bin
+modules:
+  - shared-modules/python2.7/python-2.7.json 
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/libsecret/libsecret.json
+
+
+  - name: webkitgtk 
+    config-opts:
+      - --prefix=/app
+      - --with-gtk=2.0
+      - --disable-webkit2
+    build-options:
+      cflags: -O2 -g -pipe -Wno-expansion-to-defined -Wno-class-memaccess -fno-delete-null-pointer-checks -fpermissive
+      cflags-override: true
+      cxxflags: -O2 -g -pipe -Wno-expansion-to-defined -Wno-class-memaccess -fno-delete-null-pointer-checks -fpermissive
+      cxxflags-override: true
+      env:
+        PYTHON: /usr/bin/python3
+    sources:
+      - type: archive
+        url: https://www.webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz
+        sha256: 588aea051bfbacced27fdfe0335a957dca839ebe36aa548df39c7bbafdb65bf7
+
+
+  - name: wxWidgets
+    config-opts:
+      - --with-gtk=2
+      - --enable-gui
+      - --enable-aui
+      - --disable-shared 
+      - --enable-shared=no 
+      - --enable-unicode 
+      - --disable-monolithic 
+      - --disable-debug 
+      - --disable-debug_flag 
+      - --disable-debug_gdb 
+      - --enable-propgrid 
+      - --enable-stc 
+      - --enable-html 
+      - --disable-richtext 
+      - --disable-ribbon
+      - --disable-sys-libs 
+      - --with-libjpeg=builtin 
+      - --with-libpng=builtin 
+      - --with-libtiff=builtin 
+      - --with-regex=builtin 
+      - --with-zlib=builtin 
+      - --with-expat=builtin 
+      - --with-liblzma=builtin 
+      - --with-libmspack=builti 
+      - --enable-optimise 
+      - --disable-compat28 
+      - --enable-webview
+    build-options:
+      cxxflags: "-std=c++11 -m64 "
+      env:
+        VENDOR: "alif_compiler" 
+        WX_FLAVOUR: "alif" 
+        WX_LIB_FLAVOUR: "alif_lib"
+    sources:
+      - type: git
+        url: https://github.com/wxWidgets/wxWidgets.git
+        commit: 152ec5103367d6d5e2c77ab180e9e432b58deac7
+
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-toolset=gcc --with-icu --with-python=/usr/bin/python2
+      - ./b2 install variant=release link=shared runtime-link=shared cxxflags="$CXXFLAGS" linkflags="$LDFLAGS" -j $FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2
+        sha256: 59c9b274bc451cf91a9ba1dd2c7fdcaf5d60b1b3aa83f2c9fa143417cc660722
+  
+  
+  - name: alif-lib
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/icons/hicolor/128x128/apps/ /app/usr/share/icons/hicolor/128x128/apps/ /app/share/{metainfo,applications} /app/usr/local/include/aliflibwx/ /app/usr/local/share/{aliflang,alifstudio} /app/usr/local/lib/aliflib/
```

Comment:
```suggestion
      - mkdir -p /app/share/icons/hicolor/128x128/apps/ /app/share/{metainfo,applications} /app/usr/local/include/aliflibwx/ /app/usr/local/share/{aliflang,alifstudio} /app/usr/local/lib/aliflib/
```

Reply:
as I expected 
```
Removing files/bin/idle
Removing files/bin/gtk-update-icon-cache
Removing files/bin/gtk-query-immodules-2.0
Removing files/bin/gtk-demo
Removing files/bin/gtk-builder-convert
Removing files/bin/easy_install-2.7
Removing files/bin/easy_install
Removing files/bin/2to3
Removing files/bin
Renaming org.aliflang.lang.metainfo.xml to share/appdata/org.aliflang.lang.appdata.xml
Running appstream-compose
Processing application org.aliflang.lang
Error loading desktop file: Failed to find icon: Failed to find icon org.aliflang.lang
Error: ERROR: appstream-compose failed: Child process exited with code 1
----------------------------------------------------------------------------------------
~/git/flathub(org.aliflang.lang*) »                                      1 ↵ suliman@me

```
I fixed it when installing the icon in both paths